src/middleware/rateLimiter.js

```diff
@@ -1,24 +1,17 @@
 const rateLimit = require("express-rate-limit");
-
-
-// Rate limiting
-let limiter;
-if (process.env.NODE_ENV === "development") {
-  limiter = rateLimit({
-    windowMs: 1 * 60 * 1000, // 1 minute
-    max: 3, // limit each IP to 3 requests per windowMs
-    message: "Your limit exceeded",
-    standardHeaders: true, // Return rate limit info in the `RateLimit-*` headers
-    legacyHeaders: false, // Disable the `X-RateLimit-*` headers
-  });
-} else {
-  limiter = rateLimit({
-    windowMs: 1000, // 1 second
-    max: 3, // limit each IP to 3 requests per windowMs
-    message: "Your limit exceeded",
-    standardHeaders: true, // Return rate limit info in the `RateLimit-*` headers
-    legacyHeaders: false, // Disable the `X-RateLimit-*` headers
-  });
-}
+const { MemoryStore } = require("express-rate-limit");
+ 
+const limiter = rateLimit({
+  windowMs: 1000,
+  max: 3,
+  standardHeaders: true,
+  legacyHeaders: false,
+  handler: function (req, res) {
+    return res.status(429).json({
+      error: "You sent too many requests. Please try again later",
+    });
+    },
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/armerillo/ql-task/pullRequest?prid=9976636)
```suggestion
  },
```